### ISSUE TITLE
"vsprintf(): Too few arguments" displays on every page load

### ISSUE THREAD

**Issue by qipcreative** [getsentry/magento-amg-sentry-extension#2]
The ability to output php exceptions is great, but I'm getting an error from vsprintf() in Translate.php even though the error is suppressed in that file with the @ operator.

Is there any way to make sure these suppressed errors don't get sent to Sentry, or do I have to modify the Raven php code?

**Comment by wo0dyn** [comment]
Looks pretty much like a `raven-php` issue to me… I'll check this out later.

**Comment by mateuszpohl** [comment]
I think suppressing errors is a method to hide them from users but the best way is to handle them correctly and write good code. Ability to hide errors this way is one of the worst things available in php as a language. Proper error and exceptions handling is most important. Debugging code with errors hidden like this is quite painful. I've had same error logs in sentry and I've patched Magneto itself.

**Comment by qipcreative** [comment]
Yeah I guess patching Magento is the best way to do it; I agree that it's a terrible precedent to set and it's rather irritating that Magento does it. Never mind, thank you for your help.

**Comment by LeonB** [comment]
@qipcreative I had this same issue. [I forked this repo](https://github.com/tim-online/magento-amg-sentry-extension) and fixed this issue (among other things). You could try my fork to see it fixes your problem.

**Comment by wo0dyn** [comment]
@LeonB Are you interested in managing this repo? I can add you to @getsentry/symfony team. Just let me know. :wink:

**Comment by wo0dyn** [comment]
@qipcreative : same offer btw

**Comment by geoffjukes** [comment]
@wo0dyn I'm guessing this never got fixed, as I'm having the same issue. The extension isn't ignoring @ operator errors (not just in Translate.php)
